dist/osx/parse_releases.sh

```diff
@@ -0,0 +1,80 @@
+#! /usr/bin/env node
```

Comment:
`parse_releases`, `update_release_links`, `update_releases`, `update_updates` should be moved one level above outside `osx` folder as they are not specific to macOS and can be reused for other platforms in the future. Same for `tag_version` and `upload_to_github`, but I believe these two can be removed altogether.